File: 3347-distribute-elements-into-two-arrays-i/distribute-elements-into-two-arrays-i.py
class Solution(object):
    def resultArray(self, nums):
        """
        :type nums: List[int]
        :rtype: List[int]
        """
        arr1=[]
        arr2=[]
        arr1.append(nums[0])
        arr2.append(nums[1])
        idx1=0
        idx2=0
        for i in range(2,len(nums)):
            if arr1[idx1]>arr2[idx2]:
                arr1.append(nums[i])
                idx1+=1
            else:
                arr2.append(nums[i])
                idx2+=1
        return arr1+arr2

File: 3347-distribute-elements-into-two-arrays-i/distribute-elements-into-two-arrays-i.cpp
class Solution {
public:
    vector<int> resultArray(vector<int>& nums) {
        vector<int>arr1,arr2;
        arr1.push_back(nums[0]);
         arr2.push_back(nums[1]);
         int i,j;
         int idx1=0,idx2=0;
         for(i=2;i<nums.size();i++){
            if(arr1[idx1]>arr2[idx2]){
                arr1.push_back(nums[i]);
                idx1++;
            }else{
                arr2.push_back(nums[i]);
                idx2++;
            }
         }
         vector<int>ans;
         for(i=0;i<arr1.size();i++)ans.push_back(arr1[i]);
         for(i=0;i<arr2.size();i++)ans.push_back(arr2[i]);
         return ans;
    }
};